packages/ui/src/theme/tokens/components/button.js

```diff
@@ -1,3 +1,5 @@
+const { boxShadow } = require('./menu');
```

Comment:
I don't see this being used in the file, unless I'm missing something?

Reply:
Weird, looks like an IDE autoimport mistake on my part. Will remove.